install.md

```diff
@@ -0,0 +1,22 @@
+#Install procedure
+
+To install and serve the OpenFisca France website locally :
+
+A/ Install the following packages:
+```SH
+easy_install paste
+brew install libmagic
+brew install bower
+```
+
+B/ Run the following commands:
+```SH
+python setup.py develop
```

Comment:
Why not `pip install -e .` ? I _think_ they do the same thing, but the latter is more standard.